packages/client/utils/constants.ts

```diff
@@ -46,6 +46,7 @@ export const AGENDA_ITEM = 'agendaItem'
 
 /* Sorting */
 export const SORT_STEP = 1
+export const ESTIMATE_TASK_SORT_ORDER = 2 ** 43
```

Comment:
+1 i think you've created a really great pattern here by hiding the sortOrder complexity from the client! since we'll probably use this in the future perhaps we name is something like `INTEGER_SORT_STEP` & add a comment telling future people to use this one instead of the sort_step above?

Reply:
For the same reasoning as I mentioned [below](https://github.com/ParabolInc/parabol/pull/6028#discussion_r817258215), let's leave it till we have another use case of this approach. WDYT?